### PR TITLE
fixing some basic typos for my boys

### PR DESCRIPTION
**Pull request by triwats**
* fixing some simple typos for the lads
* all your bases are belong to us

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -76,8 +77,9 @@ spec:
       prune: true
       selfHeal: true
     syncOptions:
-    - CreateNamespace=true
+      - CreateNamespace=true
```
Comment (triwats) [inline README.md:80]: This one might be vscode being pants. Please advise.

Comment (triwats) [reply README.md]: nvm having slept on it this should be fine

File: .github/workflows/merge.yaml [hunk 2]
```diff
@@ -2,29 +2,29 @@ name: Unikorn Merge
 on:
   push:
     branches:
-    - main
+      - main
```
Comment (spjmurray) [inline .github/workflows/merge.yaml:5]: Don't reformat things.  It makes reviewing changes particularly hard.  Also if you wish to enforce coding standards on things, then I'd expect an enforcement action, and for said strategy to be applied to every repository for consistency.

File: README.md [hunk 3]
```diff
@@ -5,38 +5,38 @@
...
 ## Installation
 
 ### Unikorn Prerequisites
 
-The use the Baremetal service you first need to install:
+To use the Baremetal service you first need to install:
 
-* [The identity service](https://github.com/unikorn-cloud/identity) to provide API authentication and authorization.
-* [The region service](https://github.com/unikorn-cloud/region) to provide provider agnostic cloud services (e.g. images, flavors and identity management).
+- [The identity service](https://github.com/unikorn-cloud/identity) to provide API authentication and authorization.
```
Comment (spjmurray) [inline README.md:18]: Again, this is introducing inconsistency across everything.  `*` is perfectly valid unordered list markdown and used everywhere else.  There will be times when some magical regular expression may be required across all the things and your snowflake will cause it to break/not apply unless you're actually aware (sad enough to know) of every possible permutation.

File: .github/workflows/pull-request.yaml [hunk 4]
```diff
@@ -1,64 +1,66 @@
 name: Unikorn Push
 on:
   pull_request:
+    paths-ignore:
+      - "README.md"
```
Comment (spjmurray) [inline .github/workflows/pull-request.yaml:5]: Rather than hard coding a single path, wouldn't it be a lot better to just say *.md?

**Review by spjmurray — CHANGES_REQUESTED** [review]
Tristan, Tristan ... this is literally about 5 lines of actual changes.  Why you make my life difficult 😸  That and you've managed to pick a repo that has like 2 commits, isn't finished in the slightest, doesn't even work and will in all likelihood be totally re-written and then run through aspell anyway (again in the long term this will be an action and consume a shared default custom dictionary)!  Do that instead, save me from you in the future across everything!!

File: .github/workflows/pull-request.yaml (hunk 4 above)
Comment (triwats) [reply .github/workflows/pull-request.yaml:5]: Yea let's go for that

File: .github/workflows/merge.yaml (hunk 2 above)
Comment (triwats) [reply .github/workflows/merge.yaml:5]: It's so much more readable 😢

**Comment by triwats** [comment]
@spjmurray all I'm hearing is that I am a founding contributor now. When you take over the world I'll collect my Unikorn™️ Github badge and pass go.

**Comment by spjmurray** [comment]
Screenshot from 2024-09-17 13-45-33

Green is in keeping with the branding... what do you think?  Or should I play it safe with black...

**Review by spjmurray — APPROVED** [review]
Oh go on then, couple whitespace changes, but I'll back them out shortly...

**Comment by spjmurray** [comment]
Oh dear...
> The base branch requires all commits to be signed

Get secure man, I cannot trust you are you!

**Comment by triwats** [comment]
I'd back black for the staff uniform so it goes incognito under my jumper at my part-time boxer bodyguard gig.

**Comment by triwats** [comment]
Added GPG here's my personal validation submission: 

Photo on 20-09-2024 at 14 41 #2

**Comment by spjmurray** [comment]
Like a cloud-native Jesus...

**Review by spjmurray — APPROVED** [review]

**Pull request merged**